source/publishing_data/07_configuring_visualizations/06_configuring_custom_view/custom.rst

```diff
@@ -1,109 +1,40 @@
-want (for more information about writing beautiful dashboard, see the
-:doc:`dedicated documentation </creating_content/index>`) and you can even customize the title and icon of this additional tab.
+The Custom view is available for every dataset, but is optional. It can be enabled or disabled using the tick box displayed right under the visualizations tabs.
 
-.. ifconfig:: language == 'en'
+.. screenshot
 
-   .. figure:: images/custom__view--en.png
+To set the parameters of the Custom view, follow the indications from the table below.
 
-      The custom view editor
+.. list-table::
+  :header-rows: 1
 
-.. ifconfig:: language == 'fr'
+  * * Option name
+    * Description
+    * Usage
+  * * Tab title
+    * Title of the Custom view tab in the front office. If left blank, the tab will be named "Custom view".
+    * Write a title in the Tab title textbox.
+  * * Tab icon
+    * Icon of the Custom view tab in the front office. If left blank, the icon of the tab will be the `tachometer icon <https://fontawesome.com/v4.7.0/icon/tachometer>`_.
+    * .. admonition:: Caution
+         :class: caution
 
-   .. figure:: images/custom__view--fr.png
+         Only `Font Awesome 4.7 icons <https://fontawesome.com/v4.7.0/icons/>`_ are supported as tab icon. Icons must be specified through their name, omitting the ``fa-`` prefix (e.g. to use the `bar chart <https://fontawesome.com/v4.7.0/icon/bar-chart>`_ icon, which name is ``fa-bar-chart``, write ``bar-chart`` in the Tab icon textbox).
 
-      The custom view editor
+      Write the name of the Font Awesome icon in the Tab icon textbox.
+  * * Tab URL
+    * URL to access the Custom view tab. By default, the URL to a Custom view is: ``www.domain_name.com/explore/datasets/dataset_name/custom``
+    * Write the URL of the Custom view in the Tab URL textbox.
 
-Tab icon and title
-------------------
 
-The custom title and icon will show up within your catalog page on the related dataset's card, and you'll be able to
-search for datasets having a custom view set up with the *View* filter (select the *Custom view* option).
+.. admonition:: Important
+   :class: important
 
-Icons are taken from the `Font awesome 4.7 icon set <https://fontawesome.com/v4.7.0/icons/>`_ and must be specified through
-their name (omitting the ``fa-`` prefix). For example, to use the `bar chart <https://fontawesome.com/v4.7.0/icon/bar-chart>`_
-icon, whose name is ``fa-bar-chart``, you should input ``bar-chart`` in the icon field. You can also leave this field
-empty, in which case it will default to the `tachometer icon <https://fontawesome.com/v4.7.0/icon/tachometer>`_.
+   OpenDataSoft widgets, such as the ones used for the main visualizations, can be added to a Custom view. Depending on what is the purpose of the Custom view, these widgets can either be static (i.e. independant from the rest of the page), or dynamic (i.e. able to interact with the rest of the page, for example with the dataset filters).
```

Comment:
I feel that the wording `static` is weird: in the classic sense it means static like a picture. They are dynamic in the sense that they are generated on the page load with the API. Something like "linked" or "coupled with the page"?